### ISSUE TITLE
Discovering FIM prompt template for ollama use

### ISSUE THREAD

**Issue by meditans** [TabbyML/tabby#2845]
**Please describe the feature you want**
When offloading the llm inference to `ollama` for Fill In the Middle tasks, we have to add an option like:
```
prompt_template = "<｜fim▁begin｜>{prefix}<｜fim▁hole｜>{suffix}<｜fim▁end｜>"
```
(I discovered this line, for the deepseek-coder-v2 model, from [this issue](https://github.com/TabbyML/tabby/issues/2451)).
But as I want to also try out the `codestral` like of models, I don't know what that prompt would be in that case.

Since the architectures are not a lot, it would be nice to have the `prompt_templates` for FIM for the major architectures listed in the readme.

---
Please reply with a 👍 if you want this feature.

**Comment by wsxiaoys** [comment]
https://tabby.tabbyml.com/docs/references/models-http-api/ollama/ shall be a good place to put such information

**Comment by meditans** [comment]
Indeed, thank you. Now, where would I find the authoritative information in the specific case of `codestral`? [Their documentation](https://docs.mistral.ai/capabilities/code_generation/) seems to call FIM through the API.
I found:
```
<s>[SUFFIX] {{ suffix }} [PREFIX] {{ prefix }}
```
from [here](https://github.com/ollama/ollama/issues/5403) but I'm unsure of the original provenience.
